src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+@mixin color-var() {
+  --main-color: #060b35;
+  --second-color: #616070;
+  --third-color: #00acdc;
+  --fours-color: #fff;
```

Comment:
Let's define variable using scss syntax (doing this we can remove color-var mixin)